src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -209,10 +210,22 @@
                             (flatten-row xs path))))))]
     (into {} (flatten-row row [field-name]))))
 
+(defn- type-finder
+  "Mostly just (type member) but with a bit to suss out strings which are ISO8601 and say that they are datetimes"
+  [member]
+  (let [member-type (type member)]
+    (cond
+      (= member-type java.lang.String)
+      (if (mbql-schema/can-parse-datetime? member)
+        java.time.LocalDateTime
+        member-type)
+      :else
+      member-type)))
```

Comment:
Why don't you just change this whole thing to 

```clj
(if (and (instance? String member-type)
         (mbql-schema/can-parse-datetime? member))
  java.time.LocalDateTime
  member-type)
```
    
It's the same thing and it's way more readable